kbase-extension/static/kbase/js/widgets/narrative_core/jobCommChannel.js

```diff
@@ -221,6 +254,113 @@ define([
                 case 'new_job':
                     Jupyter.notebook.save_checkpoint();
                     break;
+
+                // CELL messages
+                case 'run_status':
```

Comment:
No changes here, just moved things around to group all the `cell` messages and put the `job` messages into alphabetical order.